pkgs/by-name/ry/ryzen-monitor-ng/package.nix

```diff
@@ -14,13 +14,13 @@ stdenv.mkDerivation {
     hash = "sha256-fcW2fEsCFliRnMFnboR0jchzVIlCYbr2AE6AS06cb6o=";
   };
 
-  ## Remove binaries committed into upstream repo
+  ## Remove binaries and object files
+  ## committed into upstream repo
```

Comment:
You'd better putting "committed" in the last line to avoid ambiguity, Otherwise, others will think you are saying "I have submitted this fix to the upstream repository"

Reply:
Thank you, I have amended the comments, hopefully they are clearer now.